tools/winmd2objc/Common/WinObjC.Frameworks.UWP.Common.targets

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="_ConfirmWinObjCSDKRootEnviromentVariableIsSet" BeforeTargets="_ClangCompile">
+      <Error Condition="'$(WINOBJC_SDK_ROOT)' == ''"
```

Comment:
You can get rid of this now that you are in winobjc.
And you can remove any references to WINOBJC_SDK_ROOT